_data/chapters.yml

```diff
@@ -0,0 +1,62 @@
+# Table of Contents
+- id: intro
+  title: Introduction
+
+- id: concepts
+  title: Core Concepts
+  children:
+
+    - id: progressive-enhancement
+      title: Progressive Enhancement
+
+    - id: modularity
+      title: Modular Design
+
+    - id: web-standards
+      title: Web Standards
+
+    - id: style-guides
+      title: Style Guides
+
+    - id: frameworks
+      title: Frameworks
+
+- id: technical
+  title: Technical Guides
+  children:
+
+    - id: html
+      title: HTML
+
+    - id: css
+      title: CSS
+      children:
+
+        - id: css/architecture
```

Comment:
Since our CSS section deals with so much Sass would it be a good idea to mention it, maybe "CSS and Sass"?